frontend/src/css/Header.css

```diff
@@ -1,9 +1,9 @@
-a {
-    color: #FFF;
-    text-decoration: none;
-
-  }
-  a:hover {
-     color: #00F
-  }
+a {

```

Comment:
Instead of applying styles to entire HTML tags like this, define classes instead. 

Generally better convention to apply classes to given components/tags so that other parts of the component / app aren't affected.